### PR TITLE
CQLHS-180 | project survey welcome page

### PR DESCRIPTION
**Pull request by jonathanchaffer**

### CODE REVIEW

File: ClientApp/src/components/pages/EvaluationWelcomePage/EvaluationWelcomePage.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+  useEffect(() => {
+    setIsLoading(true);
+    // getProjectById(parseInt(id))
+    //   .then((proj) => setProject(proj))
+    //   .catch((err) => handleError(err))
+    //   .finally(() => setIsLoading(false));
+    setProject(placeholderProjectDetails);
+    setIsLoading(false);
+  }, [handleError, id]);
```
Comment (charkour) [inline ClientApp/src/components/pages/EvaluationWelcomePage/EvaluationWelcomePage.tsx:28]: Are the comments placeholder until the back end API is finished for this part?

File: ClientApp/src/components/pages/EvaluationWelcomePage/EvaluationWelcomePage.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+  return (
+    <div className="EvaluationWelcomePage">
+      {isLoading ? (
+        <Spinner animation="border" />
+      ) : (
```
Comment (charkour) [inline ClientApp/src/components/pages/EvaluationWelcomePage/EvaluationWelcomePage.tsx:34]: Mind cherry-picking again? Haha, sorry.

File: ClientApp/src/components/pages/EvaluationWelcomePage/EvaluationWelcomePage.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                    <Col md={6} xs={12}>
+                      <span>
+                        Another project is in the books! This is a quick
+                        evaluation to review the performance of your team
+                        members.
+                      </span>
+                    </Col>
+                  </Row>
+
+                  <Button size="lg">Begin Evaluation</Button>
```
Comment (charkour) [inline ClientApp/src/components/pages/EvaluationWelcomePage/EvaluationWelcomePage.tsx:59]: The Bootstrap large button isn't as large as the one's on the design comps. I don't know how picky Ally will be with QA. You can see the button [on InVision](https://projects.invisionapp.com/share/HYXPNEQ9B6D#/screens/423697477) to see if you want to update the styling.

File: ClientApp/src/placeholders/EvaluationQuestions.ts [hunk 4]
```diff
@@ -1,9 +1,8 @@
-import { ProjectRole } from "components";
 import { EvaluationQuestion } from "_generated";
 
 interface QuestionSet {
   questions: EvaluationQuestion[];
-  role: ProjectRole;
+  role: string;
```
Comment (jonathanchaffer) [inline ClientApp/src/placeholders/EvaluationQuestions.ts:5]: `role` should probably be of type `ProjectRole`, but I got a weird runtime error when doing so. This is just placeholder data so I wouldn't stress over it

File: ClientApp/src/components/reuseables/ProjectDetail/ProjectDetail.tsx [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+                    ))}
+                  </ListGroup>
+                </Card>
+              </Col>
+            </Row>
+          </Col>
+          <Col md={4} xs={12}>
+            <Row>
+              <Col>
+                <h3>Additional Information</h3>
```
Comment (charkour) [inline ClientApp/src/components/reuseables/ProjectDetail/ProjectDetail.tsx:117]: Under this section, according to the designs you need an `<h6>Dates</ h6>` above the dates and different things need to be bolded. I like that you have the important stuff in *bold* but I think you should:
- [ ] add the header for dates to match [the designs](https://projects.invisionapp.com/share/HYXPNEQ9B6D#/screens/423697477).

File: ClientApp/src/components/pages/EvaluationWelcomePage/EvaluationWelcomePage.tsx (hunk 1 above)
Comment (jonathanchaffer) [reply ClientApp/src/components/pages/EvaluationWelcomePage/EvaluationWelcomePage.tsx:28]: yep!

**Pull request merged**